views/index.njk

```diff
@@ -64,30 +65,8 @@
       text : "Total Days"
     }
   ],
-  rows: [
-    [
-      {
-        text: "01/02/2022"
-      },
-      {
-        text: "03/02/2022"
-      },
-      {
-        text: "3 Days"
-      }
-    ],
-    [
-      {
-        text: "15/10/2023"
-      },
-      {
-        text: "20/10/2023"
-      },
-      {
-        text: "5 Days"
-      }
-    ]
-  ]
+  rows: holidays
 }) }}
+{% endif %}
 
 {% endblock %}
```

Comment:
We will need to make sure that we keep one blank line at the bottom of each file. This is due to some old browsers ignoring the bottom lines.